ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/baseActivity.component.ts

```diff
@@ -122,7 +125,7 @@ export abstract class BaseActivityComponent implements OnChanges, OnDestroy {
     }
 
     protected navigateToErrorPage(): void {
-        this.router.navigateByUrl('/error');
+        this.router.navigateByUrl(this.config.errorPageUrl);
```

Comment:
A small improvement here: now apps can configure their own routes for error and dashboard pages.